src/include/inotify-cpp/Inotify.h

```diff
@@ -109,7 +113,15 @@ class Inotify {
   boost::bimap<int, fs::path> mDirectorieMap;
   int mInotifyFd;
   std::atomic<bool> mStopped;
+  int mEpollFd;
+  epoll_event mEpollEvent;
+  epoll_event mEpollEvent2;
```

Comment:
Rename mEpollEvent to mInotifyEpollEvent and mEpollEvent2 to mStopPipeEpollEvent